workflows/scrnaseq/scrnaseq-standard.wdl

```diff
@@ -89,13 +90,13 @@ workflow scrnaseq_standard {
             detect_nproc=detect_nproc
     }
     call picard.validate_bam { input: bam=count.bam, max_retries=max_retries }
-    call ngsderive.infer_strandedness as ngsderive_strandedness { input: bam=count.bam, bai=count.bam_index, gtf=gtf, max_retries=max_retries }
+    call ngsderive.infer_strandedness as ngsderive_strandedness { input: bam=count.bam, bam_index=count.bam_index, gtf=gtf, max_retries=max_retries }
 
     call md5sum.compute_checksum { input: infile=count.bam, max_retries=max_retries }
 
     output {
-        File bam = count.bam
-        File bam_checksum = compute_checksum.outfile
+        File harmonized_bam = count.bam
```

Comment:
Why `harmonized_bam`? I think the RNA-Seq workflow just uses `bam` for its output parameter. If that's a future work to update the names, then feel free to ignore this here.